### PR TITLE
Add Support for Multiple Careers and SDET + Security Engineer Roles

### PR DESCRIPTION
**Pull request by rauljordan**
This PR adds support for multiple careers in our website and adds a maintainable pattern for adding new job posts without duplicating tons of HTML. This PR also uses this new pattern to add two new roles: SDET and a Go security engineer job posting.

### CODE REVIEW

File: src/app/pages/careers-sdet/prysmatic-careers-sdet.page.component.html [hunk 1]
```diff
@@ -0,0 +1,229 @@
+
```
Comment (rauljordan) [inline src/app/pages/careers-sdet/prysmatic-careers-sdet.page.component.html:1]: Review this file

File: src/app/shared/prysmatic-header.component.html [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <div class="contract-announcement">
-  We are now hiring a frontend / UX engineer! See our careers page <a href="/careers">here</a>
+  We are now hiring for both frontend and infrastructure developer positions! See our careers page <a href="/careers">here</a>
```
Comment (terencechain) [inline src/app/shared/prysmatic-header.component.html:2]:
```suggestion
  We are now hiring for both frontend and software developer in test! See our careers page <a href="/careers">here</a>
```

File: src/app/pages/careers-sdet/prysmatic-careers-sdet.page.component.html [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+                    </li>
+                    <li>
+                        <span class="text-xl">—</span> Fully-open, permissionless, financial applications
+                    </li>
+                </ul>
+                <p class="mb-6">
+                    We believe Ethereum represents a fundamental shift in how financial and coordination systems operate. Our work is open source and accessible to anyone around the world, and we are seeking individuals that also align with these values.
+                </p>
+                <p class="mb-10">
+                    <span class="font-bold">We are looking for a remote, full-time [software development engineer in test](https://www.guru99.com/software-developer-engineer-test-sdet.html) (SDET) to keep pushing the envelope on decentralized technologies</span>. Particularly, having experience in infrastructure, devops, and deployments of software in critical functions.
```
Comment (prestonvanloon) [inline src/app/pages/careers-sdet/prysmatic-careers-sdet.page.component.html:58]: This is a markdown link, which does not render properly.
```
[software development engineer in test](https://www.guru99.com/software-developer-engineer-test-sdet.html)
```

File: src/app/pages/careers/frontend/prysmatic-careers-frontend.page.component.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import { Component } from '@angular/core';
+import { JobPost } from '../../../models/job-post';
+
+@Component({
+  selector: 'prysmatic-careers-sdet',
```
Comment (terencechain) [inline src/app/pages/careers/frontend/prysmatic-careers-frontend.page.component.ts:5]: is this selector right? shouldn't it be frontend?

Comment (rauljordan) [reply src/app/pages/careers/frontend/prysmatic-careers-frontend.page.component.ts:5]: ty yep will change

Comment (rauljordan) [reply src/app/pages/careers/frontend/prysmatic-careers-frontend.page.component.ts:5]:
```suggestion
  selector: 'prysmatic-careers-frontend',
```

**Review by prestonvanloon — APPROVED** [review]
LGTM

**Pull request merged**